lib/volumes.js

```diff
@@ -18,7 +18,16 @@ function throwInvalidSize(size) {
 
 /*
  * Returns the number of MiBs (Mebibytes) represented by the string "size". That
- * string can have different format suffixes, such as "100GB", "100G", etc.
+ * string has the following format: <integer><unit>. The integer must be > 0.
+ * Unit format suffixes are 'G' or 'g' for gibibytes and 'M' or 'm' for
+ * mebibytes. If no unit suffix is provided, the unit is considered to be
+ * mebibytes.
+ *
+ * Examples:
+ *    - the string '100' represents 100 mebibytes
+ *    - the strings '100g' and '100G' represent 100 gibibytes
+ *    - the string '100' represents 100 mebibytes
```

Comment:
Made the unit optional, and update `triton volume create`'s help as well as unit tests.